### PR TITLE
[RFR] a specific title & description for each pages

### PR DESCRIPTION
**Pull request by touv**
Now the title and description are chosen with the Syndication menu. So …
any field could be used as a title or description. In addition, we can
choose dataset title & description as the same way.

### CODE REVIEW

File: src/app/js/fields/selectors.js [hunk 1]
```diff
@@ -121,16 +120,19 @@ const getAllListFields = createSelector(getCollectionFields, fields =>
     fields.filter(f => !f.composedOf),
 );
 
-const getOverViewCol = type => fields => {
-    const result = fields.filter(f => f.overview === type);
+const findFieldWithOverviewID = id => fields => {
+    const result = fields.filter(f => f.overview === id);
```
Comment (parmentf) [inline src/app/js/fields/selectors.js:124]: Why use `filter`, while there is also `find`.
http://devdocs.io/javascript/global_objects/array/find

File: src/app/js/i18n/translations/fr.js [hunk 2]
```diff
@@ -167,6 +167,11 @@ export default {
     show_publication_errors: 'Erreurs',
     model: 'Modèle',
     overview: 'Syndication',
+    overviewNone: 'Aucun',
+    overviewTitle: 'RESSOURCE - Titre',
+    overviewDescription: 'RESSOURCE - Description',
+    overviewDatasetTitle: 'COLLECTION - Titre',
```
Comment (parmentf) [inline src/app/js/i18n/translations/fr.js]: What?!? But there is also a `cover_collection` key which translation is `Différent pour chaque ressource`. Collection is a  cover level, doesn't it?

In fact, `error_cover_invalid_cover` says `Cover must be either dataset or collection`. That means that Collection is different from Dataset.

One can't translate `Dataset` with `Collection`.

**Comment by parmentf** [comment]
Currently, the /graph page does not have a title, and thus the page's title in the browser is its URL.

**Comment by parmentf** [comment]
_RESOURCE Title_ and _RESOURCE Description_ are visible, through a badge in the ontology (view model):
image

it would be fine if the _DATASET_ side had the same badges.

File: src/app/js/fields/selectors.js (hunk 1 above)
Comment (touv) [reply src/app/js/fields/selectors.js:124]: just now, I try to use find and I don't no why but it's can not works : 

```
const findFieldWithOverviewID = id => fields =>
    fields.find(f => f.overview === id);
```

so I prefer to leave as it is

File: src/app/js/i18n/translations/fr.js (hunk 2 above)
Comment (touv) [reply src/app/js/i18n/translations/fr.js]: perhaps, the best way, is to do not translate DATASET & RESOURCE prefix in french...

File: src/app/js/fields/selectors.js (hunk 1 above)
Comment (parmentf) [reply src/app/js/fields/selectors.js:124]: But that's a `name`, that should be returned, not a field. (See line 125 below).

**Pull request merged**